lms/static/scripts/frontend_apps/components/test/SubmitGradeForm-test.js

```diff
@@ -76,6 +92,33 @@ describe('SubmitGradeForm', () => {
     assert.equal(wrapper.find('input').prop('defaultValue'), '');
   });
 
+  [
+    {
+      text: 'focuses the input field when changing students',
+      run: () => {
```

Comment:
Given the complexity of these test-case fixtures, I think this would be better as separate tests.